src/components/map/Map.tsx

```diff
@@ -60,6 +60,8 @@ class LeafletMap extends React.Component<IMapProps> {
     componentDidMount() {
         const map = this.getMap();
 
+        L.control.scale({ imperial: false }).addTo(map);
```

Comment:
Now we use `map.addControl(Control...)` and this way mixed. Might be better to stick with one.

Reply:
fixed